content/en/tracing/other_telemetry/connect_logs_and_traces/dotnet.md

```diff
@@ -346,3 +350,4 @@ Ensure that log collection is configured in the Datadog Agent and that the [Logs
 [13]: https://github.com/NLog/NLog.Extensions.Logging/wiki/NLog-properties-with-Microsoft-Extension-Logging
 [14]: https://github.com/huorswords/Microsoft.Extensions.Logging.Log4Net.AspNetCore#using-beginscope
 [15]: /logs/log_collection/csharp/#configure-your-datadog-agent
+[16]: https://github.com/DataDog/dd-trace-dotnet/tree/master/tracer/samples/AutomaticTraceIdInjection
```

Comment:
we don't use this link right?
```suggestion
```

Reply:
Hmm, yes I thought it was also used to automagically populate the `Further reading` section. Wrong.